src/components/HeroSearchProducts/SearchProducts.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { arrayIndexingWithLength } from '../../utils/arrayIndexingWithLength';
+import ProductShowcaseGrid from '../ProductShowcaseGrid';
+import Grid from '@material-ui/core/Grid';
+import { useStyles } from './SearchProducts.styles';
+import { AppBar, CssBaseline, Drawer } from '@material-ui/core';
+
+import SideBar from '../SideBar';
+import SearchBar from '../SearchBar';
+import NavBar from '../NavBar';
+const SearchProducts: React.FC = () => {
```

Comment:
The folder HeroSearchProducts/ and the file SearchProducts should be the same name